### PR TITLE
Oppdatert tekst inkl bruk som samtykkekomponent

### PR DESCRIPTION
**Pull request by Mikkelinski**
## 📥 Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## ☑️ Submission checklist

-   [ ] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [ ] `yarn build` works locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

## 💬 Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc...

### CODE REVIEW

File: packages/checkbox/README.md [hunk 1]
```diff
@@ -2,19 +2,15 @@
 
 ## Om avmerkingsbokser
 
-Du bruker avmerkingsbokser når du vil gi brukeren en liste med flere valg, der brukeren kan velge flere alternativer, alle eller ingen.
-Avmerkingsbokser kan har tre statuser
+Avmerkingsbokser brukes til å gi en liste over valg der brukeren kan velge fritt blant alternativene; en, flere, alle eller ingen. Avmerkingsbokser kan også brukes når brukeren gi sitt samtykke i en prosess.
```
Comment (Steinop) [inline packages/checkbox/README.md]: "… brukeren gir sitt samtykke …"

File: packages/checkbox/README.md [hunk 2]
```diff
@@ -2,19 +2,15 @@
...
-
-> Standard status er `Ikke valgt`.
-
-### Overskrifter til avmerkingsbokser
-
-Hvis det er nødvendig, kan du legge en overskrift over avmerkingsboksene: Det kan være en kort setning.
+-   Utilgjengelig (disabled)
 
-### Ledetekst
+Avmerkingsbokser har en ledetekst til høyre for boksen og kan ha en overskrift. Det er viktig at tekstene er tydelige og presise sånn at brukeren forstår hva som skjer dersom de markerer boksen. Standard status for avmerkingsbokser er 'Ikke valgt'.
```
Comment (Steinop) [inline packages/checkbox/README.md]: "…presise slik at brukerne forstår…" 

"Bruker" i entall eller flertall? Hvis entall: hen/han/hun. Evt:
…presise slik at brukeren forstår hva som skjer dersom boksen markeres"

File: packages/checkbox/README.md [hunk 3]
```diff
@@ -2,19 +2,15 @@
...
 
-Pass på at du bruker tydelige og presise ledetekster til avmerkingsbokser. Ledeteksten må tydelig vise hva som skjer hvis du merker boksen. Ledeteksten skal alltid stå til høyre for avmerkingsboksen.
+### Eksempler
+- Velg hva som er relevant i en liste av objekter (e.g. kategorier av mistede gjenstander ved tapt baggasje)
```
Comment (Steinop) [inline packages/checkbox/README.md]: "Bagasje"

File: packages/checkbox/README.md (hunk 1 above)
Comment (Mikkelinski) [reply packages/checkbox/README.md]: oppdatert til entall

File: packages/checkbox/README.md (hunk 3 above)
Comment (Mikkelinski) [reply packages/checkbox/README.md]: fikset

File: packages/checkbox/README.md (hunk 2 above)
Comment (Mikkelinski) [reply packages/checkbox/README.md]: entall

**Pull request merged**